packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -455,11 +450,11 @@ export function usePress(props: PressHookProps): PressResult {
         }
 
         if (e.button === 0) {
-          // Chrome and Firefox on touch Windows devices require mouse down events
-          // to be canceled in addition to pointer events, or an extra asynchronous
-          // focus event will be fired.
-          if (shouldPreventDefaultDown(e.currentTarget as Element)) {
-            e.preventDefault();
```

Comment:
note: was unable to confirm this case still, will need someone with a touch Windows device